packages/driver/src/primitives/buffer.ts

```diff
@@ -19,28 +19,50 @@
 import type char from "./chars";
 import * as chars from "./chars";
 import { LegacyHeaderCodes } from "../ifaces";
-import { Buffer } from "../adapter.node";
-
-/* WriteBuffer over-allocation */
-const BUFFER_INC_SIZE = 4096;
-
-const EMPTY_BUFFER = new Uint8Array(0);
 
 export const utf8Encoder = new TextEncoder();
 export const utf8Decoder = new TextDecoder("utf8");
 
-const decodeB64 = (b64: string): Uint8Array => {
-  return Buffer.from(b64, "base64");
-};
-const encodeB64 = (data: Uint8Array): string => {
-  const buf = Buffer.isBuffer(data)
-    ? data
-    : Buffer.from(data.buffer, data.byteOffset, data.byteLength);
-  return buf.toString("base64");
-};
+let decodeB64: (_: string) => Uint8Array;
+let encodeB64: (_: Uint8Array) => string;
+
+// @ts-ignore: Buffer is not defined in Deno
+if (typeof Buffer === "function") {
```

Comment:
Can you try this form and see if it allows us to remove the `@ts-ignore` here?

```suggestion
if (typeof globalThis.Buffer === "function") {
```

Reply:
aaa yes, globalThis!! 🎉